MatrixSDKTests/TestPlans/UnitTests.xctestplan

```diff
@@ -53,6 +53,8 @@
         "MXQRCodeDataUnitTests",
         "MXReplyEventParserUnitTests",
         "MXResponseUnitTests",
+        "MXSyncResponseUnitTests\/testMergingArrays()",
```

Comment:
Use "MXSyncResponseUnitTests" to run all of them. It can be useful in the future.
It is better to edit this file with a text editor...

Reply:
👍